### ISSUE TITLE
Does not work on Contao 4

### ISSUE THREAD

**Issue by Toflar** [codefog/contao-bootstrap#2]
Dunno if something has changed in the manager-bundle but your `Plugin` misses the 

```php
            ->setLoadAfter([
                ContaoCoreBundle::class
            ])
```

part :-)
Also, the `Enable the Bootstrap` checkbox should maybe have an `m12`, no? 😄

**Comment by qzminski** [comment]
Fixed in 59444e5.

**Comment by jacobro** [comment]
On my localhost I just installed contao/standard-edition via Composer (contao 4.3.7), then did "composer require codefog/contao-bootstrap". Everything went fine, but there is no checkbox in page layout settings and I can't use it at all. After installing Your extension, should I do anything else (update database etc.)?
